pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -1,4 +1,5 @@
-{ stdenv, fetchurl, makeWrapper, python, zip, ffmpeg, pandoc ? null }:
+{ stdenv, fetchurl, makeWrapper, pythonPackages, zip, ffmpeg
```

Comment:
The other option is to only add the `buildPythonPackage` here, however then we do not get the `python` attribute as well, and you would need to redefine both `buildPythonPackage` and `python` when doing an override of any kind.